x/bank/handler.go

```diff
@@ -55,6 +59,18 @@ func handleMsgMultiSend(ctx sdk.Context, k keeper.Keeper, msg types.MsgMultiSend
 		return types.ErrSendDisabled(k.Codespace()).Result()
 	}
 
+	for _, in := range msg.Inputs {
+		if k.BlacklistedAddrs()[in.Address.String()] {
```

Comment:
Input are _sending_ funds, not receiving them. i.e. I don't think we need the check for inputs 👍